rosetta-runtime/src/main/java/com/rosetta/model/lib/annotations/RosettaDataType.java

```diff
@@ -0,0 +1,18 @@
+package com.rosetta.model.lib.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import com.rosetta.model.lib.RosettaModelObjectBuilder;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+public @interface RosettaDataType {
+
+	String value() default "";
+	
+	Class<? extends RosettaModelObjectBuilder> builder();
+	
+	String version() default "";
```

Comment:
Got a question about this one. See Slack.

Reply:
I thought about this and I think its OK to keep this on all files that are generated.